avm/src/vm/memory.rs

```diff
@@ -208,15 +286,15 @@ impl HandlerMemory {
     if addr_type(addr) == NORMAL_ADDR {
       let addru = addr as usize;
       if self.addr.0.len() <= addru {
-        self.addr.0.resize(addru + 1, (std::usize::MAX, 0));
+        self.addr.0.resize(addru + 1, None);
```

Comment:
Can we confirm that this change is meant for child HMs to resize correctly?

Reply:
Not sure I follow. This was actually the bug fix for `Tree` and `http` since opcodes would try to read a fractal in the child and instead of going to the parent they would look at this address. However, the empty address also works for parent HMs.